### PR TITLE
Enable additional functionality for the new cloud architecture

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
Enable more functionality for the new ACLK cloud architecture (still under a feature flag)

##### Component Name
aclk,database

##### Test Plan
- The code is not active yet (it is tested with the new architecture in a separate branch)

### CODE REVIEW

File: database/sqlite/sqlite_aclk.c [hunk 1]
```diff
@@ -207,7 +189,7 @@ int aclk_start_sync_thread(void *data, int argc, char **argv, char **column)
 
 void sql_aclk_sync_init(void)
 {
-#ifdef ACLK_NEWARCH_DEVMODE
+#if ENABLE_NEW_CLOUD_PROTOCOL
```
Comment (MrZammler) [inline database/sqlite/sqlite_aclk.c:192]: Hi Stelio! Is there a reason this is `#if` instead of `#ifdef` ?

Comment (stelfrag) [reply database/sqlite/sqlite_aclk.c:192]: I will change

File: database/sqlite/sqlite_aclk.c [hunk 2]
```diff
@@ -722,19 +711,20 @@ void sql_check_aclk_table_list(struct aclk_database_worker_config *wc)
     return;
 }
 
-void aclk_data_rotated(RRDHOST *host)
+void aclk_data_rotated(void)
 {
-    UNUSED(host);
+#if ENABLE_NEW_CLOUD_PROTOCOL
```
Comment (MrZammler) [inline database/sqlite/sqlite_aclk.c:716]: This one `#if` too ?

Comment (stelfrag) [reply database/sqlite/sqlite_aclk.c:716]: yes made the changes already (won't push just yet to see if we have other changes)

File: database/sqlite/sqlite_aclk_chart.c [hunk 3]
```diff
@@ -342,6 +347,7 @@ void aclk_send_chart_event(struct aclk_database_worker_config *wc, struct aclk_d
 #else
     UNUSED(wc);
     UNUSED(cmd);
+    info("ACLK_NG not defined");
```
Comment (underhood) [inline database/sqlite/sqlite_aclk_chart.c:350]: ACLK-NG not available in this agent or sth. more friendly?

Comment (stelfrag) [reply database/sqlite/sqlite_aclk_chart.c:350]: I will either update the message or remove it .. It shouldn't even reach that point if aclk is not available

**Comment by MrZammler** [comment]
Tested this PR, built with no `ACLK_NEWARCH_DEVMODE`, claimed it to production cloud, and got an alert there for `	
system_clock_sync_state`, so that is ok.

**Comment by MrZammler** [comment]
We'll need to add this too https://github.com/netdata/netdata/blob/d0ad0cd4bad8117d36757e00bc06c79e39d0d1a2/health/health.c#L1012-L1017 (along with `aclk_alert_reloaded` variable), but I can do it in a later PR.

**Review by MrZammler — APPROVED** [review]

**Pull request merged**